patches/storejs+2.0.5.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/storejs/dist/store.js b/node_modules/storejs/dist/store.js
+index 76ebbe4..605d622 100644
+--- a/node_modules/storejs/dist/store.js
++++ b/node_modules/storejs/dist/store.js
+@@ -91,7 +91,8 @@
+       return this;
+     },
+     get: function get(key) {
+-      if (!key) {
++      // if no 'key' is passed return all
++      if (key === undefined) {
```

Comment:
we we change behavior? what is key is '' or null?

Reply:
I made this patch to fix this [bug](https://app.bugsnag.com/rudderlabs/sdk-error-reporter/errors/64fbc676966bd100086467e1).

If there is any entry in the local storage without the name, it will crash the storejs package when we count the number of entries (via the `keys` API).
With this fix, I've segregated the behavior of not sending any argument to the `get` method vs other values.

Reply:
Please also rair a PR in the storejs package it slef to rectify the same. They are acceptive to our fixes

Reply:
https://github.com/jaywcjlove/store.js/pull/45